src/modules/powerrename/PowerRenameUILib/Strings/en-us/Resources.resw

```diff
@@ -330,6 +330,12 @@
   <data name="FileCreationButton.[using:Microsoft.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
     <value>File creation date and time help</value>
   </data>
+  <data name="ErrorMessage_FileNameTooLong" xml:space="preserve">
+    <value>File name is too long</value>
```

Comment:
_could_ be a bit more specific...

Reply:
I would like to avoid numbers here. It should be fine like this as it's interactive. as soon as the user removes a few characters to fit the limit, the error is going away.